docs/General/Manuals/client-manual.rst

```diff
@@ -1480,6 +1480,171 @@ quick way to move around a large map in this manner.
+grassland+irrigation, and a grassland/river tile being worked. You can click on the city center and the client
+will automatically pick the best tiles for net food to aid city growth. You can also click on a tile to remove
+the citizen from the tile and then click another tile to have the citizen work another tile. This is commonly
+referred to a city micromanagement. If you remove a citizen from working a tile, take a look at the top center
+segment. The citizens icon bar will show one entertainer specialist. If you want to change the entertainer to
+a scientist or a taxman, you can click on it in the icon bar to change.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-center.png
+  :align: center
+  :alt: Freeciv21 City Dialog City Center
+  :figclass: align-center
+
+  Figure 48: City Dialog - City Center
+
+Moving right, we can see the :guilabel:`General` tab on the right most panel as highlighted in Figure 49. This
+tab shows information simiar to what is shown on the top center segment, along with units and city
+improvements that the city has produced and is supporting. Hovering your mouse over many of the items at the
+top of the :guilabel:`General` tab will show detailed calculation information on how the net value is
+produced.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-general.png
+  :align: center
+  :alt: Freeciv21 City Dialog General
+  :figclass: align-center
+
+  Figure 49: City Dialog - General
+
+
+We will discuss the :guilabel:`Citizens` tab in a bit. Moving to the bottom center, you will see a list of all
+the units that are present in the city as highlighted in Figure 50. There can be units present in a city that
+the city is not supporting, or from your allies. If you right-click on a unit, a pop up box shows some action
+commands that you can give to the unit. If you double-click on a unit, the :guilabel:`City Dialog` will close
+and the unit will be selected. See `Unit Controls`_ for more information on what to do with the unit.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-units.png
+  :align: center
+  :alt: Freeciv21 City Dialog Units
+  :figclass: align-center
+
+  Figure 50: City Dialog - Present Units
+
+
+Now let's come back and talk about the :guilabel:`Governor` tab on the left side segment and the
+:guilabel:`Citizens` tab on the right side segment.
+
+The :guilabel:`Governor` tab as highlighted in figure 51, gives information on the citizen management agent
+(CMA) for this city. For more information on how to use the CMA refer to :doc:`../../Playing/cma`.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-governor.png
+  :align: center
+  :alt: Freeciv21 City Dialog Govenor
+  :figclass: align-center
+
+  Figure 51: City Dialog - Govenor
+
+
+On the right side is the :guilabel:`Citizens` tab as highlighted in Figure 52. The :guilabel:`Citizens` tab
+shows you information about the happiness and the nationality of the citizens in the city. Happiness is
```

Comment:
"... shows ~~you~~ information about the happiness and ~~the~~ nationality of the citizens ~~in the city~~."